src/main/scala/GamePrompts.scala

```diff
@@ -0,0 +1,49 @@
+import GameHelpers.{calculateWordScore}
+
+object GamePrompts {
+
+  def promptName(): String = {
+    println("\u001b[33;1mWhat is your name?")
+    scala.io.StdIn.readLine().toLowerCase.capitalize
+  }
+
+  def promptLevel(): String = {
+    println("\u001b[33;1mHow good are you at Scrabble?\u001b[0m ")
+    println("[1] I'm a pro.")
+    println("[2] Go easy.")
+    println()
+    val level = scala.io.StdIn.readLine().toString
+    level match {
+      case "1" => "hard"
+      case "2" => "easy"
+      case _ => promptLevel()
```

Comment:
rather than using a String, you should use an algebraic data type - i.e.
```scala
sealed abstract class Level(val label: String)
case object Easy extends Level("easy")
case object Hard extends Level("hard")
```
Then you can pass Level instead of String, which also means that when you pattern match  in playWord, the compiler will check that you didn't miss any or mis spell anything.  And it also makes sure it didn't mix with any other strings.